Mathlib/Topology/Sheaves/LocalPredicate.lean

```diff
@@ -287,16 +287,16 @@ theorem stalkToFiber_injective (P : LocalPredicate T) (x : X)
   -- Then use induction to pick particular representatives of `tU tV : stalk x`
   obtain ⟨U, ⟨fU, hU⟩, rfl⟩ := jointly_surjective'.{v, v} tU
   obtain ⟨V, ⟨fV, hV⟩, rfl⟩ := jointly_surjective'.{v, v} tV
-  -- Decompose everything into its constituent parts:
-  dsimp
-  simp only [stalkToFiber, Types.Colimit.ι_desc_apply'] at h
-  specialize w (unop U) (unop V) fU hU fV hV h
-  rcases w with ⟨W, iU, iV, w⟩
-  -- and put it back together again in the correct order.
-  refine' ⟨op W, fun w => fU (iU w : (unop U).1), P.res _ _ hU, _⟩
-  · rcases W with ⟨W, m⟩
-    exact iU
-  · exact ⟨colimit_sound iU.op (Subtype.eq rfl), colimit_sound iV.op (Subtype.eq (funext w).symm)⟩
+  · -- Decompose everything into its constituent parts:
```

Comment:
I don't understand this change: it looks to me that there is only one goal, so no need for the focusing dot. Wrong positive of the linter, or is something fishy going on?

Reply:
You are right, this change does not belong.

This is not even a false positive: at some point, I messed up a git merge and ever since this part of the proof has always created conflicts.  The linter does not flag it and should have fixed this now!